dataservice/utils.py

```diff
@@ -13,3 +14,14 @@ def iterate_pairwise(iterable):
     a, b = tee(iterable)
     next(b, None)
     return zip(a, b)
+
+
+def to_snake_case(camel_case_str):
```

Comment:
Our resources are hyphenated, not underscored. eg `/genomic-files`, `/family-relationships`

Reply:
Yea our endpoints are hyphenated but the error messages contain our entity names which are snake cased. Example: 
```json
{
    "_status": {
        "code": 400,
        "message": "could not create family_relationship: {'participant_id': ['Missing data for required field.'], 'participant_to_relative_relation': ['Missing data for required field.'], 'relative_id': ['Missing data for required field.']}"
    }
}
```

Reply:
Right, but the snake case is still being used for the resource names in the pagination